src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const sortedStudents = JSON.parse(JSON.stringify(students));
+  const reducer = (acc: number, curt: number): number => acc + curt;
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  sortedStudents.sort((a: Student, b: Student): number => {
+    let comparator: number = 0;
+
+    switch (sortBy) {
+      case ('name'):
```

Comment:
use Enum for this
in all cases
```suggestion
      case (SortType.Name):
```